### PR TITLE
Pegasus release 2022.9

### PR DESCRIPTION
**Pull request by snopoke**
PRs:
- [x] [translations](https://github.com/czue/pegasus-docs/pull/14)

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -3,6 +3,26 @@ Version History and Release Notes
...
+
+See the new [Internationalization docs](internationalization.md) for more information options.
+
+### Other Changes
+
+- Attach team models to the request in middleware instead of view decorator
+
+### Fixes
+
+- Fix database port environment variable
```
Comment (czue) [inline release-notes.md:21]: this was already listed/included in 2022.8.1 i believe.

File: release-notes.md [hunk 2]
```diff
@@ -3,6 +3,26 @@ Version History and Release Notes
...
+
+TODO
+
+This includes support for text localization in Python, Django templates, JavaScript and Wagtail.
+
+See the new [Internationalization docs](internationalization.md) for more information options.
+
+### Other Changes
+
+- Attach team models to the request in middleware instead of view decorator
```
Comment (czue) [inline release-notes.md:17]: can you add some text around upgrading emphasizing making sure the middleware is included in settings.py? seems like the type of thing that could easily get merged incorrectly.

**Review by czue — COMMENTED** [review]
thanks! I'll probably take a pass on the overview / translations text before merging

File: release-notes.md (hunk 2 above)
Comment (snopoke) [reply release-notes.md]: 03418f4

**Pull request merged**